### ISSUE TITLE
BCALM error with conda installation: segmentation fault

### ISSUE THREAD

**Issue by taylorreiter** [spacegraphcats/spacegraphcats#258]
While @halexand and I were running spacegraphcats on a group of TARA oceans samples, we encountered a segmentation fault during the `bcalm_cdbg` rule. This error only occurs with the conda package of `bcalm`, and seems to be linked to a k-mer getting hashed to 0 (see https://github.com/GATB/bcalm/issues/31). I think this issue is fixed in BCALM commit `8a2c7c6`, but for some reason the error still occurred with the conda package. We switched to running with the master branch of BCALM and got a different error instead (#257).
Spacegraphcats log file: [serial_job_505026.log](https://github.com/spacegraphcats/spacegraphcats/files/4083603/serial_job_505026.log)
BCALM log file: [bcalm.MS-all-SRF-5-20.00.k31.unitigs.fa.log.txt](https://github.com/spacegraphcats/spacegraphcats/files/4083604/bcalm.MS-all-SRF-5-20.00.k31.unitigs.fa.log.txt)

**Comment by ctb** [comment]
is the data file available?

**Comment by halexand** [comment]
Same answer as #257 -- but this error went away when we upgraded to the BCALM master build. I think this might be more of a note from @taylorreiter than an active issue...

**Comment by taylorreiter** [comment]
Yes! This one I just wanted to record that we observed this happening.
